web-react/src/components/DisplayChurchList.jsx

```diff
@@ -12,39 +13,111 @@ const DisplayChurchList = (props) => {
+                            : null}{' '}
+                          {church.bacentas
+                            ? `${church?.bacentas?.length} Bacentas`
+                            : null}{' '}
+                          {church.constituencies
+                            ? `${church?.constituencies?.length} Constituencies`
+                            : null}{' '}
+                          {church.councils
+                            ? `${church?.councils?.length} Councils`
+                            : null}{' '}
+                          {church.streams
+                            ? `${church?.streams?.length} Streams`
+                            : null}{' '}
+                          {church.memberCount
+                            ? `| ${church?.memberCount} Members`
+                            : null}{' '}
+                          {church.vacationStatus
+                            ? `| ${church?.vacationStatus}`
+                            : null}{' '}
+                          {churchType === 'GatheringService'
+                            ? `${capitalise(church?.stream_name)}`
+                            : null}
+                        </Row>
+                      </Card.Body>
+
+                      {/* <Card.Text className="text-muted pt-2 text-small">
+                        {church.leader
+                          ? `${church.leader.firstName} ${church.leader.lastName}`
+                          : null}{' '}
+                        {church.fellowships
+                          ? `| ${church?.fellowships?.length} Fellowships`
+                          : null}{' '}
+                        {church.bacentas
+                          ? `| ${church?.bacentas?.length} Bacentas`
+                          : null}{' '}
+                        {church.constituencies
+                          ? `| ${church?.constituencies?.length} Constituencies`
+                          : null}{' '}
+                        {church.councils
+                          ? `| ${church?.councils?.length} Councils`
+                          : null}{' '}
+                        {church.streams
+                          ? `| ${church?.streams?.length} Streams`
+                          : null}{' '}
+                        {church.memberCount
+                          ? `| ${church?.memberCount} Members`
+                          : null}{' '}
+                        {church.vacationStatus
+                          ? `| ${church?.vacationStatus}`
+                          : null}{' '}
+                        {church.admin && (
+                          <p className="text-muted">{`Admin: ${church.admin.firstName} ${church.admin.lastName}`}</p>
+                        )}
+                        {churchType === 'GatheringService'
+                          ? `${capitalise(church?.stream_name)}`
+                          : null}
+                      </Card.Text> */}
```

Comment:
Please delete this if it's not being used. Don't leave it here

Reply:
alright